engine/check.py

```diff
@@ -17,38 +17,81 @@
 
 
 @click.command()
+@click.option("--input-file-ref", help=cli_help["input_file_ref"], default=None)
+@click.option("--input-file-cur", help=cli_help["input_file_cur"], default=None)
```

Comment:
I know we discused this a solution. But I think it would be nicer for the usabilt to merge `--input-file-cur` and `"--input-fof-file-cur` to `--input-files-cur`. 
```suggestion
@click.option("--input-files-cur",type=CommaSeperatedStrings(), help=cli_help["input_file_cur"], default=None)
```
So that you get a list of files, which you then can procss.
You can then process them according to their filename if it contains `fof` or `stat`.
The structure below should not change much with that.
So you would get a list of current files, ref files, and tolerance files, so it would be good to check the consistency of these files.

Reply:
It is not strictly related to want I have to do, but is it right that also in the click_util script this function is written "sepErated"?